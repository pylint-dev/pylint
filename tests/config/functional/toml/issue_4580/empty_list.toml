[tool.pylint]
# load-plugins does not belong in top level section
load-plugins = []
