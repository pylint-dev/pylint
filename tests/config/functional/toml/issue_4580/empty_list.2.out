************* Module {abspath}
{relpath}:1:0: E0014: Out-of-place setting encountered in top level configuration-section 'load-plugins' : '[]' (bad-configuration-section)
